src/main/java/uk/gov/hmcts/reform/idam/service/UserRoleService.java

```diff
@@ -59,4 +57,84 @@ public List<String> filterUsersWithRoles(List<String> staleUsers) {
             .toList();
 
     }
+
+    public void mergeRoleAssignments(String archivedUserId, String activeUserId) {
+        var response = roleAssignmentClient.getRoleAssignmentsByUserId(securityUtil.getAuthHeaders(), archivedUserId);
```

Comment:
Please rename this var to something more meaningful e.g archivedUserRoleAssignmentsResponse